doc/man3/EVP_EncryptInit.pod

```diff
@@ -375,9 +375,13 @@ exists.
 
 Encrypts I<inl> bytes from the buffer I<in> and writes the encrypted version to
 I<out>. The pointers I<out> and I<in> may point to the same location, in which
-case the encryption will be done in-place. If I<out> and I<in> point to different
-locations, the two buffers must be disjoint, otherwise the operation might fail
-or the outcome might be undefined.
+case the encryption will be done in-place. However, in-place encryption is
+guaranteed to work only if the encryption context (I<ctx>) has not been used
+before. If the context contains an incomplete data block from previous
+operations, in-place encryption may fail.
```

Comment:
I think the "has not been used before" isn't really the correct way to saying it. I suggest you just say that only multiples of the block size have been used.

Reply:
Please have a look at the rephrased text.